### DISCUSSION TITLE
Is it possible to use functions like setUpAll or tearDown in Patrol tests?

### DISCUSSION THREAD

**Discussion by Yani010712** [leancodepl/patrol#1841] [Q&A] [answered]
Is it possible to use functions like setUpAll or tearDown in Patrol tests?

**Comment by haomun0731** [comment, marked as answer]
See: https://github.com/leancodepl/patrol/issues/1895#issuecomment-1810054149

**Reply by Yani010712** [reply]
Thank you!
